adding units to fov move

adamkglaser opened the pull request:
Adding [mm] units to FOV move.

micahwoodard commented on view/widgets/acquisition_widgets/volume_model.py:
```diff
@@ -286,8 +286,8 @@ def move_fov_query(self, new_fov_pos):
-                       f"{[round(x, 2) * t for x, t in zip(self.fov_position, self.polarity)]} to "
-                       f"{[round(x, 2) for x in new_fov_pos]}?")
+                       f"{[round(x, 2) * t for x, t in zip(self.fov_position, self.polarity)]} [mm] to "
+                       f"{[round(x, 2) for x in new_fov_pos]} [mm]?")
```
Can you add a unit variable as an init argument with a default of mm, pass unit into the volume model in the volume widget and then use unit instead of hardcoding it?

micahwoodard reviewed (commented):
I think it would be good to pass the unit in as a variable, create a self.unit attribute and then use that instead of hard coding it

adamkglaser commented:
Yeah much better. Alright I think it is updated, I also removed view_color since it was deprecated. And changed default unit from um to mm in volume_widget since (I don't think it is being used in volume_widget anywhere?)

micahwoodard commented:
Yep looks good to me!
